### PR TITLE
feat: Create component to add New Issue in Kanban layout

### LINKED ISSUE #18: [Client] [Kanban] Create 'New Issue' input card/box
image

### PR DESCRIPTION
**Pull request by rajat-sr**
Resolves #18

### CODE REVIEW

**Comment by rajat-sr** [comment]
Looks like:
image

File: client/src/components/NewIssue/NewIssue.module.css [hunk 1]
```diff
@@ -0,0 +1,5 @@
+.NewIssue {
+  width: 100%;
+  margin: 10px;
+  padding: 0;
+}
```
Comment (rishabh1403) [inline client/src/components/NewIssue/NewIssue.module.css]: new line :)

File: client/src/components/NewIssue/NewIssue.module.css [hunk 2]
```diff
@@ -0,0 +1,5 @@
+.NewIssue {
```
Comment (rishabh1403) [inline client/src/components/NewIssue/NewIssue.module.css]: i think classnames use dash-case

Comment (rajat-sr) [reply client/src/components/NewIssue/NewIssue.module.css]: This is a **CSS Module** (not a CSS file). It is recommended to use camel casing in CSS Modules. 
Source: https://github.com/css-modules/css-modules#naming

**Pull request merged**